frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -112,6 +112,17 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.BAREMETAL, METAL3_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/k8s/cluster/nodes/'],
```

Comment:
@spadgett FYI - route override for the nodes page.

Reply:
Done